gamestonk_terminal/cryptocurrency/overview/overview_controller.py

```diff
@@ -0,0 +1,259 @@
+"""Cryptocurrency Overview Controller"""
+__docformat__ = "numpy"
+# pylint: disable=R0904, C0302, W0622
+import argparse
+import os
+from prompt_toolkit.completion import NestedCompleter
+from gamestonk_terminal import feature_flags as gtff
+from gamestonk_terminal.helper_funcs import get_flair
+from gamestonk_terminal.menu import session
+from gamestonk_terminal.cryptocurrency.overview import (
+    pycoingecko_view,
+    coinpaprika_view,
+)
+
+
+class Controller:
+
+    CHOICES = [
+        "?",
+        "cls",
+        "help",
+        "q",
+        "quit",
+        "cg_global",
+        "cg_defi",
+        "cg_news",
+        "cg_stables",
+        "cg_nft",
+        "cg_nft_today",
+        "cg_exchanges",
+        "cg_exrates",
+        "cg_platforms",
+        "cg_products",
+        "cg_indexes",
+        "cg_derivatives",
+        "cg_categories",
+        "cg_hold",
+        "cg_companies",
+        "cp_global",
+        "cp_markets",
+        "cp_exmarkets",
+        "cp_info",
+        "cp_exchanges",
+        "cp_platforms",
+        "cp_contracts",
+    ]
+
+    def __init__(self):
+        """CONSTRUCTOR"""
+
+        self._overview_parser = argparse.ArgumentParser(add_help=False, prog="ov")
+        self._overview_parser.add_argument("cmd", choices=self.CHOICES)
+
+    def print_help(self):
+        """Print help"""
+        print("\nOverview:")
+        print("   cls             clear screen")
+        print("   ?/help          show this menu again")
+        print("   q               quit this menu, and shows back to main menu")
+        print("   quit            quit to abandon program")
+        print("")
+
+        print("")
+        print("CoinGecko:")
+        print("   cg_global          global crypto market info")
```

Comment:
Same here for underscores_

Reply:
and same regarding
print(
print(
print(

However, again, these are not changes necessary for this PR :)